internal/github/retry_roundtripper.go

```diff
@@ -0,0 +1,41 @@
+package github
+
+import (
+	"fmt"
+	"math"
+	"net/http"
+	"time"
+)
+
+type RetryTransport struct {
+	Transport http.RoundTripper
+	Retries   int
+}
+
+func (t *RetryTransport) RoundTrip(req *http.Request) (*http.Response, error) {
+	var (
+		resp *http.Response
+		err  error
+	)
+
+	for i := 0; i <= t.Retries; i++ {
+		resp, err = t.Transport.RoundTrip(req)
+		if err != nil {
+			fmt.Println("Error:", err)
```

Comment:
Make this log similar to the one below. For example: `"Retrying request due to error: %v", err`